src/app/Events/EventTemplates.tsx

```diff
@@ -0,0 +1,65 @@
+import * as React from 'react';
+import { filter, map } from 'rxjs/operators';
+import { TextInput, Toolbar, ToolbarGroup, ToolbarItem } from '@patternfly/react-core';
+import { Table, TableBody, TableHeader, TableVariant } from '@patternfly/react-table';
+import { ServiceContext } from '@app/Shared/Services/Services';
+
+export interface EventTemplate {
+  name: string;
+  description: string;
+  provider: string;
+}
+
+export const EventTemplates = (props) => {
+  const context = React.useContext(ServiceContext);
+
+  const [templates, setTemplates] = React.useState([]);
+  const [filterText, setFilterText] = React.useState('');
+
+  const tableColumns = [
+    'Name',
+    'Description',
+    'Provider',
+  ];
+
+  const getTemplates = () => {
+    let filtered;
+    if (!filter) {
+      filtered = templates;
+    } else {
+      filtered = templates.filter((t: EventTemplate) => t.name.toLowerCase().includes(filterText) || t.description.toLowerCase().includes(filterText) || t.provider.toLowerCase().includes(filterText));
```

Comment:
I think the lowerCase setup is mixed up. Filter works for "profiling" but not "Profiling". The name is displayed as "Profiling" so I'd expect the capitalized one to work, and then ideally both to work.